.github/workflows/build-tmp.yaml

```
@@ -0,0 +1,41 @@
+
+      - name: Build & Push Image
+        uses: docker/build-push-action@v5
+        with:
+          build-args: GH_TOKEN=${{ secrets.GITHUB_TOKEN }}
+          context: .
+          cache-from: type=gha
+          cache-to: type=gha,mode=max`
+          push: true
+          platforms: linux/amd64,linux/arm64
+          tags: ghcr.io/reclaimprotocol/${{ github.event.repository.name }}:${{ github.sha }}
+          file: attestor.dockerfile
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)